jax/_src/numpy/lax_numpy.py

```diff
@@ -5567,3 +5567,34 @@ def put(a: ArrayLike, ind: ArrayLike, v: ArrayLike,
   else:
     raise ValueError(f"mode should be one of 'wrap' or 'clip'; got {mode=}")
   return arr.at[unravel_index(ind_arr, arr.shape)].set(v_arr, mode=scatter_mode)
+
+
+@util.implements(getattr(np, 'cumulative_sum', None))
+def cumulative_sum(
+    x: Array, /, *, axis: int | None = None,
```

Comment:
We should use `ArrayLike` instead of `Array` for array arguments in `jax.numpy` APIs.